_includes/up-next.html

```diff
@@ -0,0 +1,216 @@
+{% assign on_current_page = false %}
+{% assign max_steps = 3 %}
+
+<section class="module up-next">
+    <article class="module-content grid-x grid-padding-x">
+        <div class="cell small-12">
+            <h6>Up Next</h6>
+        </div>
+
+
+{% for item in site.data.pages %}
```

Comment:
This seems like quite a bit of duplication, I wonder if any of this could be factored out? I'm saying that without know what's possible via Jekyll templates though.

Reply:
How was this resolved?

Reply:
I was able to use an include, so that the html only needed to be in one place.